### PR TITLE
feat(appointments): add timezone to all emails

### PR DESCRIPTION
**Pull request by miaulalala**

### CODE REVIEW

File: lib/Service/Appointments/MailService.php [hunk 1]
```diff
@@ -217,21 +217,24 @@ private function addBulletList(IEMailTemplate $template,
...
 			$dtstart,
 			'long',
 			'short',
-			new \DateTimeZone($tzid),
+			$relativeTimezone,
 			$this->lFactory->get('calendar', $l)
 		);
 
-		$template->addBodyListItem($relativeDateTime, $l10n->t('Date:'));
+		$timestring = $relativeDateTime . ' (' . $relativeTimezone->getName() . ')';
```
Comment (tcitworld) [inline lib/Service/Appointments/MailService.php:235]: A nice future improvement we could have is using `IntlTimeZone::getDisplayName` to have the timezone name localized, if the `intl` module is available.

Comment (tcitworld) [inline lib/Service/Appointments/MailService.php:235]: Do we need to localise this? For locales that don't use brackets or invert the order of things?

Comment (miaulalala) [reply lib/Service/Appointments/MailService.php:235]: I asked Christoph and he said it won't be possible to localise it - so I agree with your future improvement idea there.

**Review by st3iny — APPROVED** [review]
Tested and works.

**Pull request merged**